composer.json

```diff
@@ -28,6 +28,7 @@
     "colinodell/json5": "^2.3",
     "doctrine/annotations": "^1.14.3",
     "guzzlehttp/guzzle": "^7.7",
+    "jumbojett/openid-connect-php": "^0.9.10",
```

Comment:
Is it ok to require the OIDC Libary via composer? The Simplesaml Class from other Plugins is not required.

Reply:
I would rather not put it there. I try to only keep the dependencies in the main composer.json that will be part of the main distribution via ZIP file, without any plugins. So I don't want to put extra weight in there for this, the SAML or Redis parts. On the servers with those plugins, I do take some extra steps therefore, by manually installing yiisoft/yii2-redis predis/predis simplesamlphp/simplesamlphp after updating the dependencies.